config/config.go

```diff
@@ -171,6 +172,13 @@ func OptionKVOpts(opts map[string]string) Option {
 	}
 }
 
+// OptionRestoreNetwork function returns an option setter for restore network
+func OptionRestoreNetwork(restore bool) Option {
```

Comment:
@mavenugo  since the daemon support live container restore only on experimental build, so this option will be set on experimental build. Setting this option will let the controller to restore `network sandbox endpoint` from the store rather than delete them on controller initialization.